cmd/dagger/cmd/do.go

```diff
@@ -86,6 +95,28 @@ var doCmd = &cobra.Command{
 	},
 }
 
+func getActions(args []string) ([][]string, error) {
+	actionsFromFlags := viper.GetStringSlice("action")
+	actions := make([][]string, 0, len(actionsFromFlags)+1)
+	for _, flagAction := range actionsFromFlags {
+		action, err := shlex.Split(flagAction)
```

Comment:
Do we need shlex?
Wouldn't a simpler `strings.Split(flagAction, " ")` be enough?
What is the use case that would need shell lexing inside of an arg?

Reply:
> Do we need shlex? [...] What is the use case that would need shell lexing?

I actually hadn't considered it, and merely tried to make sure that whatever gets passed to `--action` gets handled the same way as `args` does (lexed by the shell). However it is valid to have an action with a space in its name.

```
"do it": bash.#Run & {
	_image: alpine.#Build & { 
		packages: bash: {}
	}
	input: _image.output
	script: contents: "echo 'its done'"
}
```

`dagger do 'do it'` works here, and `dagger do -a 'do it'` should as well, but wouldn't if we weren't shlexing (I think).